docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -172,8 +151,58 @@ Column name             Type                            Description
 For tables without a table definition file, the ``_key_corrupt`` and
 ``_message_corrupt`` columns will always be ``false``.
 
+.. _kafka-table-schema-registry:
+
+Table schema and schema registry usage
+--------------------------------------
+
+The table schema for the messages can be supplied to the connector with a
+configuration file or a schema registry. It also provides a mechanism for the
+connector to discover tables.
+
+You have to configure the supplier with the ``kafka.table-description-supplier``
```

Comment:
see here @elonazoulay

Reply:
LGTM! As long as users can easily see that they have to set that config.